### DISCUSSION TITLE
(Suggestion) Radio Sending Cut Short Messages

### DISCUSSION THREAD

**Discussion by Mute-Man** [unitystation/unitystation#7523] [Ideas]
"They're in the vents! They're in the v-"
How it would work
- Begin typing Message
- Before you can finish typing, get put into crit
- Message auto-sends with a dash where you stopped typing

Simulates the button on a Radio being held down to broadcast, but then being cut short before finishing saying the message.
Would make assassinations harder to pull off though.

**Comment by PetMudstone** [comment]
Should the message also send if your headset suddenly gets taken off, such as by a traitor trying to silence you?

**Comment by PerfectTangent** [comment]
converting to discussion
